### PR TITLE
Local dev building of uikit

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

merging into a feature branch so we can get storybook fixed up in isolated PR then test using Storybook. 

- clearly separate babel / ts usage
  - ts for types, babel for building
- add `watch` command for local dev with ARGO projects
- change default exports to named exports in all files
  -  depending on build tool lots of nested default exports can be troublesome
  - added benefit explicit naming is more clean

**Type of Change**

- [ ] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- UI Kit (select one):
  - [ ] New Components with storybook stories created
  - [ ] Modified Existing components and updates stories
  - [ ] No new UIKit components or changes
- [ ] Feature is minimally responsive
- [ ] Manual testing of UI feature
- [ ] Add copyrights to new files
- [ ] Connected ticket to PR

### CODE REVIEW

File: src/VerticalTabs/index.tsx [hunk 1]
```diff
@@ -220,4 +214,4 @@ const output: typeof VerticalTabs & {
   return output;
 })();
 
-export default output;
+output;
```
Comment (demariadaniel) [inline src/VerticalTabs/index.tsx:217]: No export / return?

File: src/index.tsx [hunk 2]
```diff
@@ -7,7 +7,8 @@
  *  If not, see <http://www.gnu.org/licenses/>.
  *
  * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
- * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * EXPRESS \p0kl
+ *  o≥≥º p-jhm y0p--cxbc9t0s∂¶'OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
```
Comment (demariadaniel) [inline src/index.tsx:11]: This looks accidental?

Comment (ciaranschutte) [reply src/index.tsx:11]: Yes thank you!

File: src/VerticalTabs/index.tsx (hunk 1 above)
Comment (ciaranschutte) [reply src/VerticalTabs/index.tsx:217]: fixed this in my current branch, yes needs an export

**Comment by ciaranschutte** [comment]
@demariadaniel fixing storybook in upcoming PR
just trying to keep PRs small..... even at 89 file changes

**Review by demariadaniel — APPROVED** [review]

**Pull request merged**